app/services/sign_in/constants/error_code.rb

```diff
@@ -8,6 +8,7 @@ module ErrorCode
       LOGINGOV_VERIFICATION_DENIED = '009'
       MULTIPLE_MHV_IEN = '101'
       MULTIPLE_EDIPI = '102'
+      MULTIPLE_SEC_ID = '103'
```

Comment:
I don't see this actually being used, since an error isn't raised. I might just use the generic `Constants::ErrorCode::GENERIC_EXTERNAL_ISSUE` in the function that you pass into `check_id_mismatch`, then you don't have to define an extra error code that isn't used